distributed/tests/test_scheduler.py

```diff
@@ -1870,6 +1888,9 @@ async def test_task_groups(c, s, a, b):
     assert tg.states["forgotten"] == 5
     # Ensure TaskGroup is removed once all tasks are in forgotten state
     assert tg.name not in s.task_groups
+    assert tg._start > start
+    assert tg._stop < stop
+    assert "compute" in tg._all_durations
```

Comment:
@jrbourbeau here are cases where `@property` is needed. Cythonization of the Scheduler would result in this test failing as-is

Reply:
It looks like tests passed for the [CI build which cythonizes the scheduler](https://github.com/dask/distributed/pull/4671/checks?check_run_id=2280844688). Just to double check, should we expect this `assert` to fail, or just result in less-than-optimal cythonization?

Reply:
I'm not sure the CI is working correctly. Had tried to get it to work on Windows, but ran into issues. That was back when only GitHub Actions covered Windows. Have not revisited since getting other OSes got added to GitHub Actions